go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/ghodss/yaml v1.0.0
 	github.com/giantswarm/apiextensions/v3 v3.19.0
-	github.com/giantswarm/app/v4 v4.5.0
-	github.com/giantswarm/appcatalog v0.4.0
+	github.com/giantswarm/app/v4 v4.7.0
+	github.com/giantswarm/appcatalog v0.4.1-0.20210308105944-c53db7b6b10b
```

Comment:
Needs https://github.com/giantswarm/appcatalog/pull/59 to fix the integration tests.